backend-api/src/functions/asyncActiveSession/tokenService/tests/tokenService.test.ts

```diff
@@ -122,6 +122,130 @@ describe("Token Service", () => {
             });
           });
         });
+
+        describe("Retry policy", () => {
```

Comment:
We should use jest fake timers so we don't actually have to wait the 'real' delay for each retry

Reply:
Or delay could be an arg taken by the func that we could modify if jest timers don't play kindly with calls like this. Will investigate tomorrow